software/homebox-by-hay-kot.yml

```diff
@@ -0,0 +1,11 @@
+name: Homebox
+website_url: https://hay-kot.github.io/homebox/
+description: Homebox is the inventory and organization system built for the home user.
```

Comment:
```suggestion
description: Inventory and organization system built for the home user.
```

Reply:
Added back Go to the platforms as it is written with Go and deployed with Docker.

Reply:
>  Values for platform are the main server-side requirements for the software. Don't include frameworks or specific dialects.

Only Docker is needed

Reply:
What is the downside in having more information? It is useful to know that it is written in Go. There are multiple other listed software that includes docker and the server side language it is written in.

Reply:
Those services include deployment details for that language. 

The downside to more information is that if someone is looking to host this software, and they don't run docker, they can't use the documentation to host this software with Go.

If Go documentation is added, then Go should be added to platforms.